data/json/monster_special_attacks/weapon_spells.json

```diff
@@ -0,0 +1,145 @@
+    "type": "SPELL",
+    "name": { "str": "Pain" },
+    "description": "Increases pain.  Designed for mi-go slaver beam",
+    "valid_targets": [ "hostile" ],
+    "flags": [ "SILENT" ],
+    "//": "Listed as a recover energy effect with a negative modifier that decreases with each level of the spell, which makes it cause damage instead.",
+    "min_damage": -8,
+    "max_damage": -148,
+    "damage_increment": -14.0,
+    "max_level": 10,
+    "shape": "blast",
+    "effect": "recover_energy",
+    "effect_str": "PAIN"
+  },
+  {
+    "type": "SPELL",
+    "id": "exodii_weapon_pod",
+    "name": { "str": "Exodii multipurpose weapon pod" },
+    "description": "Randomly selects one of several weapons.",
+    "valid_targets": [ "hostile", "ground" ],
+    "effect": "none",
+    "shape": "blast",
+    "base_casting_time": 200,
+    "min_damage": 1,
+    "max_damage": 1,
+    "extra_effects": [
+      { "id": "weaponpod_electric_blast" },
+      { "id": "weaponpod_heat_gun" },
+      { "id": "weaponpod_saw" },
+      { "id": "weaponpod_saw" },
+      { "id": "weaponpod_saw" },
+      { "id": "weaponpod_saw" }
+    ]
+  },
+  {
+    "id": "weaponpod_electric_blast",
+    "type": "SPELL",
+    "name": { "str": "Electric Arc Burst" },
+    "description": "The machine emits an explosion of arcing electricity.",
+    "effect": "attack",
+    "shape": "cone",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "LOUD" ],
+    "max_level": 3,
+    "min_damage": 20,
+    "max_damage": 60,
+    "damage_increment": 20.0,
+    "min_range": 5,
+    "max_range": 7,
+    "range_increment": 1,
+    "min_aoe": 10,
+    "max_aoe": 30,
+    "aoe_increment": 10.0,
+    "base_casting_time": 200,
+    "field_id": "fd_electricity",
+    "min_field_intensity": 1,
+    "max_field_intensity": 1,
+    "field_chance": 1,
+    "sound_description": "a loud crack",
+    "damage_type": "electric"
+  },
+  {
+    "id": "weaponpod_heat_gun",
+    "type": "SPELL",
+    "name": { "str": "Combat Heat Gun" },
+    "description": "The machine emits a blast of searing heat.",
+    "effect": "attack",
+    "shape": "cone",
+    "valid_targets": [ "hostile", "ground" ],
+    "flags": [ "LOUD", "IGNITE_FLAMMABLE" ],
+    "max_level": 3,
+    "min_damage": 20,
+    "max_damage": 60,
+    "damage_increment": 20.0,
+    "min_range": 3,
+    "max_range": 5,
+    "range_increment": 1,
+    "min_aoe": 20,
+    "max_aoe": 60,
+    "aoe_increment": 20.0,
+    "base_casting_time": 200,
+    "sound_description": "a whoosh",
+    "damage_type": "heat"
+  },
+  {
+    "id": "weaponpod_saw",
+    "type": "SPELL",
+    "name": { "str": "Combat Saw Arm" },
+    "description": "The machine lashes out with a spinning sawblade on some kind of extension arm.",
+    "effect": "attack",
+    "shape": "line",
+    "valid_targets": [ "hostile", "ground" ],
+    "max_level": 3,
+    "min_damage": 40,
+    "max_damage": 80,
+    "damage_increment": 20.0,
+    "min_range": 1,
+    "max_range": 2,
```

Comment:
Does it pick effects that don't have the range to reach it's target?

Reply:
Dunno! probably. I am going to have to test it out.
I might shorten all the effects to fairly close range, though.